CHANGELOG.md

```diff
@@ -3,7 +3,61 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.35.0
+
+This release contains a number of fixes and enhancements.
+
+### Early Exit Optimization
+
+This release adds an early exit optimization to the evaluator. With this optimization, the evaluator stops evaluating rules when an answer has been found and subsequent evaluation would not yield any new answers. The optimization is automatically applied to complete rules and functions that meet specific requirements. For more information see the [Early Exit in Rule Evaluation](https://www.openpolicyagent.org/docs/edge/policy-performance/#early-exit-in-rule-evaluation) section in the docs. [#2092](https://github.com/open-policy-agent/opa/issues/2092)
```

Comment:
We can probably use `latest` for docs rather than `edge` since this will be released when most people read it, right?

Reply:
Yep, I just lookup the content in the edge docs and then copy paste the URL from the browser. Forgot to update the links manually. Fixed now.